recipes/codem/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "codem" %}
+{% set version = "0.23.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/codem-{{ version }}.tar.gz
+  sha256: 21f609590b11d6e552ecf62c7872bc200f3e6b2e191eb0527e9a55ee512f8950
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - codem = codem.main:main
+    - vcd = vcd.main:main 
+
+requirements:
+  build:
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}     # [linux]
+  host:
+    - xorg-libxfixes  # [linux]
+    - python
+    - pip
+  run:
+    - python
+    - typing-extensions
+    - numpy
+    - python-pdal
+    - opencv
+    - rasterio
+    - trimesh
+    - matplotlib-base
+    - scikit-image
+    - rich
+    - pyproj
+    - pyshp
+    - pandas
```

Comment:
Both `pyproject.toml` and `setup.cfg` do not declare the dependencies but I did find the following with depfinder:

```
 'required': ['codem',
              'matplotlib',
              'numpy',
              'opencv',
              'pandas',
              'pdal',
              'pyproj',
              'pyyaml',
              'rasterio',
              'rich',
              'scikit-image',
              'scipy',
              'shapefile',
              'trimesh',
              'typing_extensions',
              'vcd']}
```

Some are there, some are not (like scipy and pyyaml) and vcd seems to be shipped with codem. You are the experts on this package here, should you add the missing ones?

Reply:
The target environment of this tool has been conda-environments; and due to the dependence of `python-pdal` (only available via `conda-forge`) I never felt obligated to list of the runtime dependencies inside pyproject.toml/setup.cfg/setup.py.  We do provide an [environment.yml file](https://github.com/NCALM-UH/CODEM/blob/main/environment.yml).

The other wrinkle is that we want to give users the option of running opencv-python-headless (not available on conda-forge).

That said, scipy and pyyaml should absolutely be listed 😬

Reply:
> The target environment of this tool has been conda-environments; and due to the dependence of python-pdal (only available via conda-forge) I never felt obligated to list of the runtime dependencies inside pyproject.toml/setup.cfg/setup.py. We do provide an [environment.yml file](https://github.com/NCALM-UH/CODEM/blob/main/environment.yml).

B/c the package is on PyPI it would be nice to follow Python packaging standards. You don't need to fix that right now to get this merged but it can be done in future iterations. The beauty of it is that our automated tools can parse the metadata and create/update PRs for you in conda-forge.